### DISCUSSION TITLE
SOUP hangs

### DISCUSSION THREAD

**Discussion by sblanch** [Security-Onion-Solutions/securityonion#8835] [General]
I'm trying to run SOUP to upgrade to .170, but it just seems to hang (see below).  It has been sitting like this for the past few hours. I have never had this issue before.  I have restarted security onion, and tried again, but it did not help...

This node can communicate with the salt-master.
Checking to see if this is a manager.

This is a manager, so we can proceed.

The manager's pillars can be rendered. We can proceed with SOUP.

Checking to see if this is an airgap install.

Found that Security Onion 2.3.160 is currently installed.

Cloning Security Onion github repo into /tmp/sogh/securityonion.
Removing previous upgrade sources.
Cloning the Security Onion Repo.
Cloning into 'securityonion'...
remote: Enumerating objects: 60575, done.
remote: Counting objects: 100% (908/908), done.
remote: Compressing objects: 100% (383/383), done.
Receiving objects:   0% (22/60575), 20.01 KiB | 0 bytes/s

**Comment by dougburks** [comment]
Do you have a firewall or other network device that may be interfering with the git clone?

**Comment by sblanch** [comment]
I use PiHole for DNS, but I do not see any queries from security onion that have been blocked.  I have not changed anything there (but it doesn't generate it's block lists dynamically).  I can whitelist git if necessary. 

So I just opened PiHole to watch for blocked queries and ran SOUP, and SOUP is running just fine this time.  Changed nothing. I'll keep monitoring it.
